internal/pkg/test/fixtures/fixtures.go

```diff
@@ -118,6 +118,21 @@ var trustZoneFixtures map[string]*trust_zone_proto.TrustZone = map[string]*trust
 		Federations:         []*federation_proto.Federation{},
 		AttestationPolicies: []*ap_binding_proto.APBinding{},
 	},
+	// tz5 has no federations or bound attestation policies and has an external SPIRE server.
```

Comment:
It might be good to add an explicit `false` to the commonly used tz1 or tz2 since that is how it will be defined in the data store.